.github/workflows/snap-release.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - name: Build snap
         run: |
-          sudo apt-get install -y git git-extras
+          sudo apt-get install -y git git-extras pkg-config libssl-dev
```

Comment:
Nit: Packages not in alphebetical order ;) Same comment for below.

Reply:
Thanks, I'll fix it